src/Symfony/Component/Messenger/Tests/Middleware/TraceableMiddlewareTest.php

```diff
@@ -140,4 +141,92 @@ public function handle(Envelope $envelope, StackInterface $stack): Envelope
         $traced->handle($envelope, new StackMiddleware(new \ArrayIterator([null, $middleware])));
         $this->assertSame(1, $middleware->calls);
     }
+
+    public function testClonedTraceableStackUnstacksIndependently()
+    {
+        // import TraceableStack
+        class_exists(TraceableMiddleware::class);
```

Comment:
@nicolas-grekas, I added this test method to test if a clone can be unstacked independently. I noticed that the existing method `testHandle()` in the same file does test `TraceableStack` unstacking via the use of `TraceableMiddleware`, but it doesn't test if a cloned version will unstack the stack independently, so I was thinking their could be a new test for that as well?

Reply:
Right, without it I get the following error: 
`Error: Class 'Symfony\Component\Messenger\Middleware\TraceableStack' not found`

To fix the error, I tried to import `TraceableStack` at the top of the test file, but I can't get it to work because it's located within `TraceableMiddleware.php` instead of within its own file.